docs/products/opensearch/howto/restore-snapshot-opensearch.md

```diff
@@ -0,0 +1,300 @@
+  --bucket my-bucket \
+  --region us-west-2 \
+  --access-key AWS_ACCESS_KEY \
+  --secret-key AWS_SECRET_KEY \
+  --endpoint https://s3.amazonaws.com
+```
+
+**Parameters:**
+
+- `PROJECT_NAME`: The name of your Aiven project.
+- `SERVICE_NAME`: The name of your Aiven for OpenSearch service.
+- `repository`: The name of the snapshot repository.
+- `repository-type`: The repository type, set to `s3` for Amazon S3 and
+  S3-compatible services.
+- `bucket`: The S3 bucket name where the snapshot is stored.
+- `region`: The AWS region where the bucket is located.
+- `access_key`: The AWS access key.
+- `secret_key`: The AWS secret key.
+- `endpoint`: (Optional) The service endpoint for S3-compatible services.
+
+</TabItem>
+</Tabs>
+
+### Google Cloud Storage (GCS)
+
+For Google Cloud Storage (GCS), use one of the following methods to register the
+snapshot repository
+
+<Tabs groupId="config-methods">
+<TabItem value="api" label="API" default>
+
+```bash
+curl -X PUT "https://SERVICE_NAME.aivencloud.com/_snapshot/my-snapshot-repo" \
+-H "Authorization: Bearer API_TOKEN" \
+-H "Content-Type: application/json" \
+-d '{
+  "type": "gcs",
+  "settings": {
+    "bucket": "my-gcs-bucket",
+    "base_path": "snapshots/",
+    "credentials": "GCS_CREDENTIALS_FILE_CONTENT"
+  }
+}'
+```
+
+**Parameters:**
+
+- `SERVICE_NAME`: The name of your Aiven OpenSearch service.
+- `API_TOKEN`: Your API token for authentication.
+- `bucket`: The GCS bucket name where the snapshot is stored.
+- `base_path`: The path within the bucket where the snapshot is stored.
+- `credentials`: The GCS credentials file content.
+
+</TabItem>
+<TabItem value="cli" label="CLI">
+
+```bash
+avn service elasticsearch-register-repository \
+  --project PROJECT_NAME \
+  --service SERVICE_NAME \
+  --repository my-snapshot-repo \
+  --repository-type gcs \
+  --bucket my-gcs-bucket \
+  --base-path snapshots/ \
+  --credentials GCS_CREDENTIALS_FILE_CONTENT
+```
+
+**Parameters:**
+
+- `PROJECT_NAME`: The name of your Aiven project.
+- `SERVICE_NAME`: The name of your Aiven OpenSearch service.
+- `repository`: The name of the snapshot repository.
+- `repository-type`:The repository type, set to `gcs` for Google Cloud Storage.
+- `bucket`: The GCS bucket name where the snapshot is stored.
+- `base_path`: The path within the bucket where the snapshot is stored.
+- `credentials`: The GCS credentials file content.
+
+</TabItem>
+</Tabs>
+
+### Microsoft Azure
+
+For Microsoft Azure, use one of the following methods to register the snapshot repository.
+
+<Tabs groupId="config-methods">
+<TabItem value="api" label="API" default>
+
+```bash
+curl -X PUT "https://SERVICE_NAME.aivencloud.com/_snapshot/my-snapshot-repo" \
+-H "Authorization: Bearer API_TOKEN" \
+-H "Content-Type: application/json" \
+-d '{
+  "type": "azure",
+  "settings": {
+    "account": "AZURE_ACCOUNT_NAME",
+    "container": "my-container",
+    "base_path": "snapshots/",
+    "key": "AZURE_SECRET_KEY",
```

Comment:
same account and key are not part of the PUT repository request

Reply:
removed this